FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/include/FreeRTOS_IP_Private.h

```diff
@@ -59,6 +59,44 @@ extern "C" {
 #define ipDECL_CAST_PTR_FUNC_FOR_TYPE( TYPE ) TYPE * vCastPointerTo_##TYPE( void * pvArgument )
 #define ipDECL_CAST_CONST_PTR_FUNC_FOR_TYPE( TYPE ) const TYPE * vCastConstPointerTo_##TYPE( const void * pvArgument )
 
+
+/* Pointer conversion to BaseType_t */
+static portINLINE ipDECL_CAST_PTR_FUNC_FOR_TYPE( BaseType_t )
```

Comment:
I suspect that the conversions to `BaseType_t *` and `TickType_t *` are probably not needed, can you try removing these and just using normal casting?

Reply:
Removed these casting functions. Most of the casts are from void * to BaseType_t thereby falling under advisory rule 11.5